rotkehlchen/tests/fixtures/blockchain.py

```diff
@@ -108,12 +109,13 @@ def fixture_kusama_manager(
     Consider mocking task manager, related with issue #2057: do not let task
     manager schedule any tasks during tests by default.
     """
-    manager = SubstrateManager(
+    kusama_manager = SubstrateManager(
         chain=SubstrateChain.KUSAMA,
         msg_aggregator=messages_aggregator,
         greenlet_manager=greenlet_manager,
     )
-    return manager
+    wait_until_all_substrate_nodes_connected(substrate_manager=kusama_manager)
```

Comment:
This should be configurable, we don't want connections to be made in every test that is run, no?